### ISSUE TITLE
[Accessibility][Functional - Settings -Onboarding]: No action is performed on activating Onboarding 'Enabled' toggle button.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2101]
**User Experience:**
For keyboard users it is difficult to perform the task if any control does not perform its functionality.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19569.1000)​​
Microsoft Edge Version: 82.0.436.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Settings' from left navigation.
5. Select 'Onboarding' button available under 'Configuration'.
6. Observe the issue on activating Onboarding 'Enabled' toggle button.

**Actual Result:**
No action is performed on activating Onboarding 'Enabled' toggle button. As it is mentioned in text that 'Enabling Onboarding will restart the product tour' but no actioned is performed.

**Expected Result:**
Action should be performed on activating Onboarding 'Enabled' toggle button.

Attachment:

[Functional_Settings_Onboarding.zip](https://github.com/microsoft/BotFramework-Composer/files/4255200/Functional_Settings_Onboarding.zip)

**Comment by corinagum** [comment]
I don't believe this scenario prompts a restart of the app. My understanding is that onboarding is re-enabled, and prompts will show up again on the UI. (when navigating to other parts of the app)

I believe this trigger should have an screenreader announcement saying it is enabled or disabled, but there is no need to cause a refresh. 

Could a developer confirm the expected behavior here? :) In terms confirming visually the change, I'm not quite sure what the ideal solution would be.

Would it be beneficial to have a toast as a confirmation that the setting has changed? This would provide extra visual feedback, and provide a placeholder of the aria-live text for the screenreader to announce the change in state.

**Comment by beyackle** [comment]
I'll take a stab at this - I see something that should be useful here, although I'll want a designer to take a look at it to confirm it works well and looks good.

**Comment by beyackle** [comment]
![image](https://user-images.githubusercontent.com/61990921/77707886-bd90be00-6f83-11ea-9109-b2ffa796f227.png)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
